wherehows-web/app/constants/dataset-aclaccess.js

```diff
@@ -0,0 +1,111 @@
+import _ from 'lodash';
+
+/**
+ * Defined the method returns the page static content
+ * @param {string} userName
+ * @return {Object} pageContent
+ */
+const pageContent = userName => {
+  userName = _.capitalize(userName);
```

Comment:
I think Ember comes with it's own capitalize function attached to strings already in case you don't want to have to import underscore from lodash

Reply:
agreed